Lib/configparser.py

```diff
@@ -18,9 +18,9 @@
              delimiters=('=', ':'), comment_prefixes=('#', ';'),
              inline_comment_prefixes=None, strict=True,
              empty_lines_in_values=True, default_section='DEFAULT',
-             interpolation=<unset>, converters=<unset>):
-
-        Create the parser. When `defaults` is given, it is initialized into the
+             interpolation=<unset>, converters=<unset>,
+             allow_unnamed_section=False):
+        Create the parser. When `defaults' is given, it is initialized into the
```

Comment:
This change looks to be incorrect. Matching backticks are normally used for identifiers. Actually, the single-backticks are probably incorrect for ReStructuredText, but that's a separate issue. Probably better to retain the original backtick for consistency.

```suggestion
        Create the parser. When `defaults` is given, it is initialized into the
```